packages/common/src/groups/_internal/GroupSchema.ts

```diff
@@ -11,14 +11,14 @@ export type GroupEditorType = (typeof GroupEditorTypes)[number];
 export const GroupEditorTypes = [
   "hub:group:edit",
   "hub:group:settings",
-  "hub:group:discussions",
```

Comment:
Removing line 14 changes all the other subsequent values' indices which is related to the test failures you're observing. I would recommend replacing the original line 14 with the line you added on 21